### ISSUE TITLE
Propose conforming to ISO standards document format / structure 

### ISSUE THREAD

**Issue by doudulac** [Open-Mobility-Hub/wheelchair-digital-interface#11]
If the ultimate goal is to propose WDI as an ANSI/ISO standard, I recommend starting now to structure the documentation accordingly. I don't think it's necessary to replace the markdown headings currently in use, and the formalism dictated by the standards document structure aren't very intuitive for a readme.md. But maybe we could think about adding the mandatory sections, or at the very least, think about which markdown section headers would map to them. What lead me to this comment was a lack of term definitions. The term "device", for example, is used quite often, and while we have an idea of what that is, it will need at least a definition, if not a newly defined, non-generic term. The following are the mandatory sections I could identify from the simple template provided by ISO. 

1.  Scope
2.  Normative references 
3.  Terms and definitions 

The ' Introduction ', while not listed as mandatory, seems a good spot for Background and Purpose. Just some thoughts before we have to restructure and rewrite sections. Been there, done that. Not fun. 

https://www.iso.org/iso-templates.html
[Download the simple draft template for ISO standards](https://www.iso.org/sd/fetch/D4mWwRVG9n03FI4zKaeMO8JqioWi5I5yGKeJLKgeU7SUsHDg-o9hc9DbG3_aeio7) (.doc)
http://www.iso.org/iso/model_document-rice_model.pdf

**Comment by jeredhdean** [comment]
Agreed. As much as possible we should format for future use. Adding this as a #TODO

**Comment by jeredhdean** [comment]
Closing this issue, no plans for RESNA/ISO submission at this time.
